.github/workflows/build-and-test.yml

```diff
@@ -382,16 +382,25 @@ jobs:
         run: |
           set -eux
           date
-          juju config microceph snap-channel=reef/candidate
+          target="reef/stable"
+          juju config microceph snap-channel=$target
           sleep 60
           # Initial wait
           juju wait-for application microceph --query='forEach(units, unit => unit.workload-status=="active" && unit.agent-status=="idle")' --timeout=30m || true
           # Get status
           juju ssh microceph/0 -- sudo ceph -s
+          juju ssh microceph/0 -- sudo microceph cluster list
           juju status
           # Fail if goal status not reached
           juju wait-for application microceph --query='forEach(units, unit => unit.workload-status=="active" && unit.agent-status=="idle")' --timeout=2m
           date
+          # Test if we have expected snap releases
+          juju exec -a microceph -- sudo snap info microceph
```

Comment:
Is this line needed?

Reply:
This was for informational purposes -- i.e. if the test in the next line fails we could scroll back and see which snap versions were actually active

Reply:
Figured that was the case.